src/actions/guides/testing/html_and_interactivity.cr

```diff
@@ -7,21 +7,38 @@ class Guides::Testing::HtmlAndInteractivity < GuideAction
 
   def markdown : String
     <<-MD
-    ## Setup
+    ## About Flow
+
+    LuckyFlow is a shard that allow you to programmatically control a browser. This is best used
+    to test your front-end and how user's directly interact with your website in a browser.
+
+    The name "flow" comes from the idea that a user will follow a "flow" from some starting point
+    to some ending point. (e.g. Clicking login, entering login details, and being redirected to a dashboard)
+
+    LuckyFlow is automatically installed and configured with Lucky full (or web) projects.
+    API based projects don't need it since there's no HTML, CSS, and JavaScript.
 
-    LuckyFlow is automatically installed and configured with Lucky projects
-    (excluding projects generated with the `—api` option)
+    ## Setup
 
-    For tests to run you will need to install Chrome and Chromedriver:
+    LuckyFlow uses [chromedriver](https://chromedriver.chromium.org/) to control Selenium under the hood.
+    You do not need to install anything as the LuckyFlow shard will handle that for you.
 
-    ### On macOS
+    The LuckyFlow configuration settings are located in `spec/setup/configure_lucky_flow.cr`.
 
-    `brew cask install chromedriver`
+    ```crystal
+    # spec/setup/configure_lucky_flow.cr
+    LuckyFlow.configure do |settings|
+      settings.stop_retrying_after = 200.milliseconds
+      settings.base_uri = Lucky::RouteHelper.settings.base_uri
+      settings.retry_delay = 10.milliseconds
 
-    ### On Linux
+      # Change where screenshots are stored
+      settings.screenshot_directory = "./tmp/screenshots"
 
-    `sudo apt-get install chromium-chromedriver` or
-    https://makandracards.com/makandra/29465-install-chromedriver-on-linux
+      # Change which browser you use (i.e. Brave, Firefox, Edge, etc...)
+      settings.browser_binary = "/Applications/Brave Browser.app/Contents/MacOS/Brave Browser"
```

Comment:
```suggestion
      # Point to a different Chrome browser than the default
      settings.browser_binary = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
```

We only support Chrome at the moment https://github.com/luckyframework/lucky_flow/issues/91